dspace-spring-rest/src/test/java/org/dspace/app/rest/RootRestResourceControllerIT.java

```diff
@@ -34,25 +34,25 @@ public void listDefinedEndpoint() throws Exception {
-                   .andExpect(jsonPath("$._links.metadataschemas.href", startsWith(REST_SERVER_URL)))
-                   .andExpect(jsonPath("$._links.resourcePolicies.href", startsWith(REST_SERVER_URL)))
-                   .andExpect(jsonPath("$._links.sites.href", startsWith(REST_SERVER_URL)))
-                   .andExpect(jsonPath("$._links.submissiondefinitions.href", startsWith(REST_SERVER_URL)))
-                   .andExpect(jsonPath("$._links.submissionforms.href", startsWith(REST_SERVER_URL)))
-                   .andExpect(jsonPath("$._links.submissionsections.href", startsWith(REST_SERVER_URL)))
-                   .andExpect(jsonPath("$._links.submissionuploads.href", startsWith(REST_SERVER_URL)))
-                   .andExpect(jsonPath("$._links.workspaceitems.href", startsWith(REST_SERVER_URL)))
-                   .andExpect(jsonPath("$._links.authn.href", startsWith(REST_SERVER_URL)))
+                   .andExpect(jsonPath("$._links.authorities.href", startsWith(BASE_REST_SERVER_URL)))
+                   .andExpect(jsonPath("$._links.bitstreamformats.href", startsWith(BASE_REST_SERVER_URL)))
+                   .andExpect(jsonPath("$._links.bitstreams.href", startsWith(BASE_REST_SERVER_URL)))
+                   .andExpect(jsonPath("$._links.browses.href", startsWith(BASE_REST_SERVER_URL)))
+                   .andExpect(jsonPath("$._links.collections.href", startsWith(BASE_REST_SERVER_URL)))
+                   .andExpect(jsonPath("$._links.communities.href", startsWith(BASE_REST_SERVER_URL)))
+                   .andExpect(jsonPath("$._links.epersons.href", startsWith(BASE_REST_SERVER_URL)))
+                   .andExpect(jsonPath("$._links.groups.href", startsWith(BASE_REST_SERVER_URL)))
+                   .andExpect(jsonPath("$._links.items.href", startsWith(BASE_REST_SERVER_URL)))
+                   .andExpect(jsonPath("$._links.metadatafields.href", startsWith(BASE_REST_SERVER_URL)))
+                   .andExpect(jsonPath("$._links.metadataschemas.href", startsWith(BASE_REST_SERVER_URL)))
+                   .andExpect(jsonPath("$._links.resourcePolicies.href", startsWith(BASE_REST_SERVER_URL)))
+                   .andExpect(jsonPath("$._links.sites.href", startsWith(BASE_REST_SERVER_URL)))
+                   .andExpect(jsonPath("$._links.submissiondefinitions.href", startsWith(BASE_REST_SERVER_URL)))
+                   .andExpect(jsonPath("$._links.submissionforms.href", startsWith(BASE_REST_SERVER_URL)))
+                   .andExpect(jsonPath("$._links.submissionsections.href", startsWith(BASE_REST_SERVER_URL)))
+                   .andExpect(jsonPath("$._links.submissionuploads.href", startsWith(BASE_REST_SERVER_URL)))
+                   .andExpect(jsonPath("$._links.workspaceitems.href", startsWith(BASE_REST_SERVER_URL)))
+                   .andExpect(jsonPath("$._links.authn.href", startsWith(BASE_REST_SERVER_URL)))
```

Comment:
Why is this test changed to check only for `BASE_REST_SERVER_URL` now?

Reply:
We identified a problem with identifying the REST API URI when used behind a HTTP proxy which grants HTTPS access.
We’re working to move this to a separate PR, it’s not specific to the entities

Reply:
This has been moved to https://github.com/DSpace/DSpace/pull/2258
Once in master, it can be removed here